src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -376,4 +204,199 @@ public void teleopPeriodic() {
       emergencyDriveCommand.cancel();
     }
   }
+
+  private enum AutoMode {
+
+    LEFT_DOUBLE_SCALE_THEN_SWTICH("Left Double Scale Then Switch", new DecisionNode(new DecisionNode[]{
+        new DecisionNode(Posession.SCALE_OWNED_LEFT, MotionProfile.LEFT_DOUBLE_SCALE_THEN_SWITCH),
+        new DecisionNode(Posession.SCALE_OWNED_RIGHT, new DecisionNode[]{
+            new DecisionNode(Posession.SWITCH_OWNED_LEFT, MotionProfile.LEFT_HOOK),
+            new DecisionNode(Posession.SWITCH_OWNED_RIGHT, MotionProfile.GO_STRAIGHT)
+        }),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
+    })),
+    LEFT_SCALE_THEN_SWITCH("Left Scale Then Switch", new DecisionNode(new DecisionNode[]{
```

Comment:
Since the names are 1:1 corresponded to what displays on the dashboard maybe just nix the name field and use the name of the constant (IIRC enum `toString()`s by default return the name).

Reply:
Eh, I normally do that for my enums, but this was how you had it set up and the flexibility isn't hurting anyone.

Reply:
Considering we want to be able to define new autos fast, maybe add an overloaded constructor that defaults?